presto-server-rpm/src/main/resources/dist/config/catalog/localfile.properties

```diff
@@ -0,0 +1,10 @@
+#
```

Comment:
Please remove this comment, as this config is being shipped with the production RPM.